### DISCUSSION TITLE
Import Multiple Jobs

### DISCUSSION THREAD

**Discussion by casey1701** [jhuckaby/Cronicle#480] [Q&A]
I have about 20 jobs I add to cronicle every once in awhile. Right now I am copying each one and editing some details. Would it be possible for me to create a script, or a list of jobs that I could find/replace to edit in notepad and inject into cronicle somehow?

**Comment by jhuckaby** [comment]
Have you checked out the [REST API](https://github.com/jhuckaby/Cronicle#json-rest-api), specifically the [create_event](https://github.com/jhuckaby/Cronicle#create_event) call?  It might do what you want.

If not, I will look into adding the ability to bulk-import a JSON file or something.

**Comment by dheise-rheinenergie-trading-com** [comment]
Today I used a little bash script with curl to create about 320 new events using the create_event call. Works really well. 

I looped over my csv file, took a json template for a new event, replaced my placeholders with the real data and called the service. Pretty simple.
